src/core/plugin/luapi_application.h

```diff
@@ -811,6 +812,139 @@ static int applib_addStrokes(lua_State* L) {
     return 0;
 }
 
+/**
+ * Puts a Lua Table of the Strokes (from the selection tool / selected layer) onto the stack.
+ * Is inverse to app.addStrokes
+ *
+ * Required argument: type ("Selection" or "Layer")
```

Comment:
We should be consistent with respect to lower case/upper case.

Please change "Selection" and "Layer" to lower case.